users.js

```diff
@@ -95,6 +96,86 @@ var lockedUsers = Object.create(null);
 var lockedRanges = Users.lockedRanges = Object.create(null);
 var rangelockedUsers = Object.create(null);
 
+function importLocksBans() {
+	fs.readFile('config/locksbans.csv', function (err, data) {
```

Comment:
I'm still not sure how I think about `locksbans.csv` as the name. Maybe something like `punishments.csv`.